controllers/pinController.js

```diff
@@ -8,6 +8,14 @@ const findMainPins = async(req, res) => {
     return res.status(200).json({data : pinInfos});
 }
 
+const findRecommendPins = async(req, res) => {
+    const pageSize = req.query.pageSize ? Number(req.query.pageSize) : 0;
+    const page = req.query.page ? Number(req.query.page) : 0; 
+    const { tagId } = req.params;
+    const recommendPinInfos = await pinService.getRecommendPins(tagId, pageSize, page);
```

Comment:
```suggestion
    const recommendPinInfos = await pinService.getRecommendPins(tagId, pageSize = 0, page);
```
default 주고 싶으시면 이렇게 줄 수 있을겁니다.

Reply:
해당 사항 반영하여 전체 핀 조회 코드에도 수정하였습니다.